app/src/main/java/org/simple/clinic/widgets/PrimarySolidButton.kt

```diff
@@ -0,0 +1,74 @@
+package org.simple.clinic.widgets
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.graphics.RectF
+import android.graphics.drawable.ColorDrawable
+import android.graphics.drawable.Drawable
+import android.support.v7.widget.AppCompatButton
+import android.util.AttributeSet
+import android.util.DisplayMetrics
+import org.simple.clinic.R
+import timber.log.Timber
+
+/**
+ * Accepting better names.
+ *
+ * TODO: Draw bottom border.
+ * TODO: Draw touch feedback.
+ */
+class PrimarySolidButton(context: Context, attrs: AttributeSet?) : AppCompatButton(context, attrs) {
+
+  private val dpToPx = { dp: Int ->
```

Comment:
Shouldn't this be a function? I don't see any benefit in declaring it as a lambda?